### ISSUE TITLE
Support for Guardrails for Amazon Bedrock

### ISSUE THREAD

**Issue by brnaba-aws** [langchain-ai/langchain-aws#25]
Guardrails for Amazon Bedrock since GA has changed the payload when using boto3.
This is the new payload content:

```
response = client.invoke_model(
    body=b'bytes'|file,
    contentType='string',
    accept='string',
    modelId='string',
    trace='ENABLED'|'DISABLED',
    guardrailIdentifier='string',
    guardrailVersion='string'
)
```

Changes in BedrockBase are needed. I'll also create a PR

**Comment by brnaba-aws** [comment]
This is the PR : https://github.com/langchain-ai/langchain-aws/pull/26

**Comment by brnaba-aws** [comment]
fix in #26

**Comment by brnaba-aws** [comment]
Which version of boto3 are you using?
